fetch.bs

```diff
@@ -3736,6 +3794,9 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
     </ol>
   </ol>
 
+ <li><p>If the <a>cross-origin resource policy check</a> with <var>request</var> and
```

Comment:
@youennf if we fixed #741 and skipped this step if the `CORS flag` is set, we'd address the CORS-preflight issue I think.